src/gql/queries/dataset.ts

```diff
@@ -43,6 +43,13 @@ export const GET_DATASET_SCHEMAS = gql`
     scene(projectId: $projectId) {
       id
       datasetSchemas(first: $first, last: $last, after: $after, before: $before) {
+        edges {
+          node {
+            id
+            source
+            name
+          }
+        }
```

Comment:
There is a chance the format of data being retrieved changed when pagination was implemented on the backend. If this edges/node is necessary now, perhaps the nodes {id souce name } below isn't needed anymore. Please double check and remove if unneeded.

Reply:
removed

Reply:
It doesn't seem to be removed yet?